### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by shreyasgosavi**
Normal un-layered styles have higher precedece over normal-Layered style. Previously it was written that normal-unlayered styles have higer precesence over normal-important, which is incorrect since layers-styles marked with important have higher precedence than unlayered style.

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ The above code creates two named layers and one unnamed layer. Let us assume tha
 
 Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. Normal styles in the first created layer have the lowest precedence, and normal styles in the layer created last have the highest precedence among the layers. In other words, normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.
 
-Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Normal styles not declared in a layer always have higher precedence than normal-importance layered styles.
+Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Normal styles not declared in a layer always have higher precedence than normal-layered styles.
```
Comment (chrisdavidmills) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:336]: I think the original wording was correct — it is saying that non-layer styles have higher precedence than normal-importance layered styles, and by "normal-importance", I think it means "not set as `!important`".

However, I do get that the wording is confusing. Can I suggest that you update it to something like this:

> Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Styles not declared in a layer always have higher precedence than styles that _are_ declared inside a layer (with the exception of [`!important`](/en-US/docs/Web/CSS/important) styles).

What do you think?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@shreyasgosavi thanks a lot for your contribution to MDN! I think the original wording was correct but confusing. See below for more details, and a suggestion. It's great that you have helped us identify an issue here.

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md (hunk 1 above)
Comment (shreyasgosavi) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md:336]: Hello  @chrisdavidmills  !! Thank you for your response ... I am still not getting the explanation for the original-one, but as you have suggested the changes which makes it more clear and easy to understand I am updating the request . 

Again thanks for your time .

Comment (chrisdavidmills) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md]: Great, thanks for the updates @shreyasgosavi!

I have taken this opportunity to make a few small grammar fixes to the same page, but your changes look good. I will now approve and merge.

**Review by chrisdavidmills — APPROVED** [review]
LGTM

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md (hunk 1 above)
Comment (shreyasgosavi) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md]: Thanks Chris !!!

**Pull request merged**